apps/application-system/api/src/app/modules/payment/e2e/payment.spec.ts

```diff
@@ -21,7 +22,33 @@ class MockPaymentApi {
       receptionID: faker.datatype.number(),
     }
   }
-
+  async getCatalogByPerformingOrg() {
```

Comment:
I think this might not be used anymore. I removed the call from being used, so you probably don't need to bring it back